addOns/scripts/src/main/zapHomeFiles/scripts/templates/extender/Add toolbar button.js

```diff
@@ -70,8 +71,8 @@ function create_window() {
   menubar.add(menu);
   window.setJMenuBar(menubar);
   var lbl = new jlabel("A Label");
-  lbl.setHorizontalAlignment(jlabel.CENTER);
-  lbl.setVerticalAlignment(jlabel.CENTER);
+  lbl.setHorizontalAlignment(SwingConstants.CENTER);
+  lbl.setVerticalAlignment(SwingConstants.CENTER);
```

Comment:
This works without being declared as var/Java type? Cool

Reply:
It was declared.

Reply:
Doh, I must have expanded the wrong section or something 🤦‍♂️